frontend/src/app/[locale]/research/ResearchImpact.tsx

```diff
@@ -64,12 +63,6 @@ const ResearchImpact = () => {
             ),
             strong: (chunks) => <strong>{chunks}</strong>,
             subscribe: (chunks) => <a href={"/subscribe"}>{chunks}</a>,
-            arrowUpRightFromSquare: () => (
```

Comment:
Removed b/c the link is not local. This also doesn't need to be an icon, the link could just include the `usa-link--external` class.